### PR TITLE
feat: add mashup examples of QCS and QSEoW for demo

### PR DESCRIPTION
**Pull request by LiKang6688**
Providing an example of mashup working within QCS
using build tool
without build tool - It has the ability to run sn-table which is not released yet

Providing an example of mashup working within QSEow

### CODE REVIEW

File: mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+*.log
```
Comment (haxxmaxx) [inline mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/.gitignore:1]: All of these shouldn't be needed

**Comment by haxxmaxx** [comment]
Why the capital letters in the file names?

File: mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+import connect from './connect';
+// import embed from './configure';
+import './style.css';
+
+(async () => {
+  const app = await connect();
+  // const nuked = embed(app);
+  const nuked = embed(app, {
+    context: {
```
Comment (haxxmaxx) [inline mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/index.js:12]: Why is this here and in the configuration?

File: mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/index.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                align: 'left',
+              },
+            },
+          },
+        ],
+      },
+    },
+  });
+
+  // .then((viz) => {
```
Comment (haxxmaxx) [inline mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/index.js:62]: Should this be removed?

File: mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/.gitignore (hunk 1 above)
Comment (LiKang6688) [reply mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/.gitignore:1]: okay, removing this .gitignore file?

File: mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/configure.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { embed } from '@nebula.js/stardust';
+import table from '@nebula.js/sn-table/core/esm/index';
+
+const nuked = embed.createConfiguration({
+  context: {
+    theme: 'dark',
+    language: 'zh-CN',
```
[the remark this answers is not in the document]
Comment (LiKang6688) [reply mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/configure.js:7]: right :joy:

File: mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/.gitignore (hunk 1 above)
Comment (haxxmaxx) [reply mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/.gitignore:1]: No I mean I don't think we need to ignore all of this

**Comment by LiKang6688** [comment]
> Why the capital letters in the file names?

folder name? let me rename them

File: mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/index.js (hunk 2 above)
Comment (LiKang6688) [reply mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/index.js:12]: Cleaned up

File: mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/index.js (hunk 3 above)
Comment (LiKang6688) [reply mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/index.js:62]: Cleaned up

File: mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/.gitignore (hunk 1 above)
Comment (LiKang6688) [reply mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/.gitignore:1]: okay

**Comment by haxxmaxx** [comment]
> > Why the capital letters in the file names?
> 
> folder name? let me rename them
Yeah the only the name of the folders actually

**Comment by haxxmaxx** [comment]
I have a refactor idea. The two examples using parcel seem to have more incommon that the two desktop examples, package json and html look identical, no? Put those two in one folder and have the non-build tool example in a different folder

**Comment by LiKang6688** [comment]
> > I have a refactor idea. The two examples using parcel seem to have more incommon that the two desktop examples, package json and html look identical, no? Put those two in one folder and have the non-build tool example in a different folder
> 
> Okay, i have no opionion about it. But sure.

Done

**Comment by haxxmaxx** [comment]
My point with the refactor was that you could reuse at least the package and html file, but might be easier to keep them separate if they diverge in the future

**Review by haxxmaxx — APPROVED** [review]

**Comment by LiKang6688** [comment]
> My point with the refactor was that you could reuse at least the package and html file, but might be easier to keep them separate if they diverge in the future

yaah, i think that they will diverge in the future
